Makefile.am

```diff
@@ -76,15 +76,15 @@ CHECK_FLAGS2  = [quick],[standard]
 
 check_PROGRAMS = lstest
 
-lstest_SOURCES =  tests/main.test.cc
+lstest_SOURCES =  tests/main.test.cc      tests/bmat.test.cc
```

Comment:
This isn't in the correct place. Do you care?

Reply:
Yes, probably

Reply:
Oh wait you mean `main.test.cc` is in the wrong place, I think we should leave it there

Reply:
I agree with leaving the ``main.test.cc`` where it is, but then the ``bmat.test.cc`` is still placed inconsistently relative to the others